articles/user-guide/libraries/standard/error-correction.md

```diff
@@ -51,15 +56,15 @@ To see how we can identify that this is the case without measuring the very stat
 | $X_1$ | $\ket{010}$ | $\ket{101}$ |
 | $X_2$ | $\ket{001}$ | $\ket{110}$ |
 
-In order to protect the state that we're encoding, we need to be able to distinguish the three errors from each other and from the identity $\boldone$ without distinguishing between $\ket{\overline{0}}$ and $\ket{\overline{1}}$.
-For example, if we measure $Z_0$, we get a different result for $\ket{\overline{0}}$ and $\ket{\overline{1}}$ in the no-error case, so that collapses the encoded state.
-On the other hand, consider measuring $Z_0 Z_1$, the parity of the first two bits in each computational basis state.
-Recall that each measurement of a Pauli operator checks which eigenvalue  the state being measured corresponds to, so for each state $\ket{\psi}$ in the table above, we can compute $Z_0 Z_1 \ket{\psi}$ to see if we get $\pm\ket{\psi}$.
-Note that $Z_0 Z_1 \ket{000} = \ket{000}$ and that $Z_0 Z_1 \ket{111} = \ket{111}$, so that we conclude that this measurement does the same thing to both encoded states.
+To protect the state that you're encoding, you need to be able to distinguish the three errors from each other and the identity $\boldone$ without distinguishing between $\ket{\overline{0}}$ and $\ket{\overline{1}}$.
+For example, if you measure $Z_0$, you get a different result for $\ket{\overline{0}}$ and $\ket{\overline{1}}$ in the no-error case, so that collapses the encoded state.
```

Comment:
It looks like in this article, $Z_0$ refers to `PauliZ` on the first qubit and `PauliI` elsewhere, but in the above article, that was $Z_1$. Can we perhaps take the opportunity to normalize the two a little bit? Relatedly, it may help to provide a concrete call to the Q# `Measure` operation here.